src/ServiceArgumentInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @file
+ * Provides Drupal\services\ServiceArgumentInterface.
+ */
+
+namespace Drupal\services;
+use Symfony\Component\HttpFoundation\Request;
+
+interface ServiceArgumentInterface {
```

Comment:
This can be blown away - we replaced this with the ServiceArgument annotation and now has no functional relevance